src/__tests__/legalEntityManagement.spec.ts

```diff
@@ -0,0 +1,326 @@
+        });
+    });
+
+    describe("Documents", (): void => {
+        it("should support POST /documents", async (): Promise<void> => {
+            scope.post("/documents")
+                .reply(200, document);
+
+            const request: DocumentUpdate = {
+                attachments: [{
+                    content: "string",
+                    contentType: "string",
+                    filename: "string",
+                    pageName: "string",
+                    pageType: "string"
+                }],
+                description: "document",
+                owner: {
+                    "id": "123456789",
+                    "type": "passport"
+                },
+                type : DocEnum.DriversLicense
+            };
+
+            const response: models.Document = await legalEntityManagement.Documents.create(request);
+
+            expect(response.id).toBe(id);
+            expect(response.type).toBe(DocEnum.DriversLicense);
+            expect(response.owner).toEqual({id : "123456789", type : "passport" })
+        });
+
+        it("should support GET /documents/{id}", async (): Promise<void> => {
+            scope.get(`/documents/${id}`)
+                .reply(200, document);
+
+            const response: models.Document = await legalEntityManagement.Documents.retrieve(id);
+
+            expect(response.id).toBe(id);
+            expect(response.type).toBe(DocEnum.DriversLicense);
+            expect(response.owner).toEqual({id : "123456789", type : "passport" })
+        });
+
+        it("should support PATCH /documents/{id}", async (): Promise<void> => {
+            scope.patch(`/documents/${id}`)
+                .reply(200, document);
+
+            const request: DocumentUpdate = {
+                attachments: [{
+                    content: "string",
+                    contentType: "string",
+                    filename: "string",
+                    pageName: "string",
+                    pageType: "string"
+                }],
+                description: "document",
+                owner: {
+                    "id": "123456789",
+                    "type": "passport"
+                },
+                type : DocEnum.DriversLicense
+            };
+
+            const response: models.Document = await legalEntityManagement.Documents.update(id, request);
+
+            expect(response.id).toBe(id);
+            expect(response.type).toBe(DocEnum.DriversLicense);
+            expect(response.owner).toEqual({id : "123456789", type : "passport" })
+        });
+
+        it("should support DELETE /documents/{id}", async (): Promise<void> => {
+            scope.delete(`/documents/${id}`)
+                .reply(200);
+
+            await legalEntityManagement.Documents.delete(id);
+
+        });
+    });
+
+    describe("OnboardingLinks and Themes", (): void => {
+        it("should support POST /onboardingLinks", async (): Promise<void> => {
```

Comment:
```suggestion
        it("should support POST /legalEntities/{id}/onboardingLinks", async (): Promise<void> => {
```

Reply:
Actually shouldn't it be best to include it since this is in fact the whole link? https://docs.adyen.com/api-explorer/#/legalentity/v2/post/legalEntities/{id}/onboardingLinks